style.css

```diff
@@ -1010,7 +1010,7 @@ Structure
 	text-align: center;
 }
 .site-logo {
-	width: 100%;
+	width: auto;
```

Comment:
here I'm not sure if I remember correctly, and this was a problem. But i think there might be a possibility to be a problem with the logo on safari/mac. Could you please just try and see that everything is ok? 👍